Aquifer/NSFileHandleExt.m

```diff
@@ -50,9 +45,9 @@ + (instancetype)aqu_fileHandleWithConnectionToHost:(NSString *)host toPort:(int)
 
 	struct sockaddr_in remoteaddr;
 	bzero((char *)&remoteaddr, sizeof(remoteaddr));
-	remoteaddr.sin_family = AF_INET;
+	remoteaddr.sin_family		= AF_INET;
```

Comment:
I generally don't like the look of aligned code in declarations unless it's repeating a bunch of the same declaration with only minor identifier changes.  But it is growing on me... It's so minor I think it can stay.